res/controllers/lib/mididispatcher.mjs

```diff
@@ -19,7 +28,7 @@ export class MidiDispatcher {
     receiveData(data, timestamp) {
         const key = JSON.stringify([data[0], data[1]]);
         const callback = this.inputMap.get(key);
-        if (typeof callback === 'function') {
+        if (callback !== undefined) {
             callback(data, timestamp);
```

Comment:
Can you explain this change to me? What other callable non-function objects are there supposed to be?

Reply:
This change shouldn't make any difference because setInputCallback now verifies that it receives a function for the callback. However, JS doesn't have private members, so it is possible that something other than a function could be stored in inputMap. Therefore, I changed it back.

Reply:
Ahh. I was missing that context. Thanks for clarifying and changing it back.